src/main/webapp/app/entities/course/course.component.html

```diff
@@ -68,22 +68,34 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                                 ><span class="bold">{{ 'artemisApp.course.shortName' | translate }}:</span> {{ course.shortName }}</span
                             >
                         </td>
-                        <td>
+                        <td class="d-none d-md-table-cell">
                             <span class="bold">{{ 'artemisApp.course.students' | translate }}:</span> {{ course.studentGroupName }}<br />
                             <span class="bold">{{ 'artemisApp.course.tutors' | translate }}:</span> {{ course.teachingAssistantGroupName }}<br />
                             <span class="bold">{{ 'artemisApp.course.instructors' | translate }}:</span> {{ course.instructorGroupName }}
                         </td>
                         <td>{{ course.startDate | date: 'medium' }}</td>
                         <td>{{ course.endDate | date: 'medium' }}</td>
-                        <td>
+                        <td class="d-none d-md-table-cell">
                             <span *ngIf="course.onlineCourse">{{ 'artemisApp.course.onlineCourseTrue' | translate }}</span>
                             <span *ngIf="!course.onlineCourse">{{ 'artemisApp.course.onlineCourseFalse' | translate }}</span>
                         </td>
-                        <td>
+                        <td class="d-none d-md-table-cell">
                             <span *ngIf="course.presentationScore !== 0">{{ 'global.generic.yes' | translate }}</span>
                             <span *ngIf="course.presentationScore === 0">{{ 'global.generic.no' | translate }}</span>
                         </td>
-                        <td class="text-right">
+                        <td jhiIdeFilter [showInIDE]="true">
```

Comment:
please double check if you can remove this, because the exercises button already exists

Reply:
I checked and I think it is better to leave this, because the regular exercise button is part of a `td` that also contains all other buttons. I can just remove the whole `td` in IntelliJ if I leave the filter as is. Otherwise, I would have to add a filter to every `button` in the `td`

Reply:
ok